### PR TITLE
feat(datepicker): vue3 style and keyboard navigation enhancement #95

### PR DESCRIPTION
**Pull request by iropolo**
# Vue3 Datepicker style and keyboard navigation enhancement

Vue2:  #95

### CODE REVIEW

**Review by francisrupert — CHANGES_REQUESTED** [review]
Each button in Previous Month/Year row should be `muted` style.

image

**Comment by iropolo** [comment]
> Each button in Previous Month/Year row should be `muted` style.
> 

Right, seems like I forget to move more styles. Now I just directly use `d-btn--muted` class.

Updated in this vue3 PR on commit https://github.com/dialpad/dialtone/pull/98/commits/64ab02ba6f5c2ca76d2190850a3c96f8a2716b77

Updated also in vue2 PR #95 commit https://github.com/dialpad/dialtone/pull/95/commits/619731221b45bcc2933755c8d3844a57dec5f233

File: packages/dialtone-vue3/components/datepicker/modules/month-year-picker.vue [hunk 1]
```diff
@@ -1,73 +1,132 @@
...
-        :aria-label="`${changeToLabel} ${prevMonthLabel} ${formattedMonth(selectMonth - 1, MONTH_FORMAT)}`"
-        @click="changeMonth(-1)"
-        @keydown="handleKeyDown($event)"
+        <template #anchor>
+          <dt-button
+            id="prevYearButton"
+            :ref="el => { if (el) setDayRef(el) }"
+            size="xs"
+            importance="clear"
+            class="d-btn--icon-only d-datepicker__nav-btn d-btn--muted"
```
Comment (francisrupert) [inline packages/dialtone-vue3/components/datepicker/modules/month-year-picker.vue]: Should be able to do...
```
size="xs"
importance="clear"
kind="muted"
circle
class="d-datepicker__nav-btn"
```

Comment (iropolo) [reply packages/dialtone-vue3/components/datepicker/modules/month-year-picker.vue]: 👍🏼 
Added 
```
kind="muted"
circle
```
and clered class to `class="d-datepicker__nav-btn"` 

Updated in this vue3 PR on commit 
[03bcd03](https://github.com/dialpad/dialtone/pull/98/commits/03bcd033cff7daa9b6a0dc324e82e2fe6462778e)

PR #95 vue2 commit https://github.com/dialpad/dialtone/pull/95/commits/760a0f3eb3862521934b5ffb0d25c91d1c5c023d

**Review by braddialpad — APPROVED** [review]

**Pull request merged**